### ISSUE TITLE
[DOCUMENTATION] System Architecture

### ISSUE THREAD

**Issue by awpala** [omshub/website#40]
Once established, document the overall system architecture, including third party services used, access instructions, etc.

**Comment by CBlumini** [comment]
This seems like something that someone newer could help out with. Would be a good opportunity to talk to more experienced devs and get an understanding of how something like this gets put together. Thoughts?

**Comment by disposedtrolley** [comment]
Absolutely! Something simple and high level enough to be resilient against change would be perfect in this stage. We can even look at programmatically creating the diagram using [Mermaid](https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/) and committing it to a `docs/` subfolder in this repo.
